lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/classify/PercentileClassifier.java

```diff
@@ -9,21 +9,19 @@
  * Returns a new dataframe with a column representation the classification status for
  * each row: 1.0 if outlier, 0.0 otherwise.
  */
-public class PercentileClassifier implements Transformer {
+public class PercentileClassifier extends Classifier {
     // Parameters
     private double percentile = 0.5;
     private boolean includeHigh = true;
     private boolean includeLow = true;
-    private String columnName;
-    private String outputColumnName = "_OUTLIER";
 
     // Calculated values
     private double lowCutoff;
     private double highCutoff;
     private DataFrame output;
```

Comment:
Yeah, each classifier should save the column name it outputs results in so I've moved it into the base class. getResults will change for each implementation though: some classifiers might cache it and some might want to compute stuff on demand.